src/inmanta/parser/__init__.py

```diff
@@ -46,7 +46,7 @@ def export(self) -> ast_export.Error:
 class ParserWarning(InmantaWarning, ParserException):
     """Warning occurring during the parsing of the code"""
 
-    def __init__(self, location: Range, value: object, msg: str) -> None:
+    def __init__(self, location: Location, value: object, msg: str) -> None:
```

Comment:
This is fine right ? Rationale is that in the safe_decode function, I want to raise a `ParserWarning` but I only have access to enough info for a `Location` (i.e. file name and line number) and not enough for a `Range` (i.e. start and end positions).
As far as I can tell, the argument is only ever used as a `Location`. Am I missing anything ?

Reply:
I think you're right.